packages/react-context-selector/src/utils.ts

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react'
+
+export const CONTEXT_SUBSCRIBE_PROPERTY = 's'
```

Comment:
I might suggest looking into `Symbol.for('ContextSubscribeProperty`) it will help to ensure that the Symbol is unique to that text without the cost of passing that string around.

Reply:
As a note, that's how react identifies it's components.

Reply:
Although, IE doesn't support it.

Reply:
I will remove these constants until we will have a thing that measures bundle size